src/main/java/org/candlepin/resource/ActivationKeyResource.java

```diff
@@ -205,6 +205,15 @@ public ActivationKeyDTO addPoolToKey(
         Long quantity) {
 
         ActivationKey key = this.fetchActivationKey(activationKeyId);
+
+        // If we're running in SCA mode, don't allow adding (more) pools to the keys,
+        // because owner should see all the pools anyway
+        Owner owner = key.getOwner();
+        if (owner != null && owner.isUsingSimpleContentAccess()) {
+            throw new BadRequestException(i18n.tr("Pools cannot be added to activation keys while" +
```

Comment:
Double indent on the next line, and there's no space between "while" and "the" on the carry-on.